contrib/docker/tester/Dockerfile

```diff
@@ -22,13 +22,13 @@ RUN cd /opt && \
     git clone https://github.com/dealii/candi && \
     cd candi && \
     mv /opt/local.cfg . && \
-    ./candi.sh -p /opt -j64 && \
+    ./candi.sh -p /opt -j2 && \
     rm -rf /opt/tmp && \
     ln -s /opt/astyle-2.04/astyle /usr/bin/astyle
 
 # Set environment variables for this image to be used
 # by Github Actions
-ENV DEAL_II_DIR /opt/deal.II-v9.4.0
+ENV DEAL_II_DIR /opt/deal.II-master
```

Comment:
Does it make sense to keep this folder intact or maybe introduce subfolders for the different versions?

Reply:
This is the name candi chooses inside the install directory. I do not know if I can change that.

Reply:
I am sorry, I meant "does it make sense to keep ``contrib/docker/tester/Dockerfile`` as is or introduce a 9.4 and a master version"?

Reply:
Ah I see. Well they are pretty identical and easy to change. I would prefer not creating another copy that can get out of date.